UIComponents/Sources/UIComponents/ViewControllers/AlertViewController.swift

```diff
@@ -43,7 +43,7 @@ public final class AlertViewController: UIViewController {
             button = factory.makeAccentInvertedButton()
         }
 
-        button.condifure(
+        button.configure(
```

Comment:
the same change I saw in that PR https://github.com/AnimealProject/animeal_iOS/pull/54
Do we need it here ?

Reply:
it's fixes build error